CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 
 ## Unreleased
 
-* Add border option to breadcrumb ([PR #1952])(https://github.com/alphagov/govuk_publishing_components/pull/1952) MINOR
+* Add border option to breadcrumb ([PR #1952](https://github.com/alphagov/govuk_publishing_components/pull/1952) MINOR
```

Comment:
I think it's still missing a closing `)` at the end 🤔

Reply:
Argh, yes it is. Thank you!